### ISSUE TITLE
webtor.io loads js from malicious domain

### ISSUE THREAD

**Issue by quinncomendant** [webtor-io/web-ui-v1#2]
I noticed https://webtor.io/ tries to load a javascript file from `https://redoutcomecomfort[.]com/60/1b/f3/601bf3ec83cfebfef8038f451009400d.js`. The domain `redoutcomecomfort[.]com` is flagged as [malicious 2/92](https://www.virustotal.com/gui/url/02c517dd7b3825e2f38583a48f8350f7fd24ab2a7997ce303ab157b603cb0226) on VirusTotal. The request for this file is initiated by `webpack:///ui/src/lib/store/actions.js`, the URL of which appears to come from your ads provider. So, this isn't so much an issue with webtor as it is with the ads provider you've chosen.

[Here is a HAR file](https://github.com/webtor-io/web-ui/files/8575712/webtor.io.har.zip) for the page load of webtor.io that includes the request to this domain.

![Screen Shot 2022-04-27 at 13 08 10](https://user-images.githubusercontent.com/6884276/165600505-b8438e41-33c2-4503-bd0f-02c31573ecd9.png)

**Comment by vintikzzz** [comment]
Yes, you are right, it's my ad provider for pops. In case if you use self-hosted solution, no ad scripts are loaded.
